### ISSUE TITLE
Add toggle to remove arrows from vanilla Fleet Logs

### ISSUE THREAD

**Issue by stormbringer951** [stormbringer951/CaptainsLog#76]

**Comment by jaghaimo** [comment]
You could replace objects of `BreadcrumbIntel` class with `MyBreadcrumbIntel` objects which extends `BreadcrumbIntel` and overrides `getArrowData()`. Messy and ugly :(

**Comment by stormbringer951** [comment]
> You could replace objects of `BreadcrumbIntel` class with `MyBreadcrumbIntel` objects which extends `BreadcrumbIntel` and overrides `getArrowData()`. Messy and ugly :(

Yeah, it's something I'm resistant to but having the arrows is *really* ugly.

I suppose I can add it as an optional extension feature defaulted off; would probably have to implement some way of replacing old BreadcrumbIntel items if I want to uninstall. Would just have to be very careful that I only selectively replace intel items I mean to.

**Comment by stormbringer951** [comment]
> If you do replacement route, I'd do it via composition AND inheritance + Lombok delegates.

Huh. Not played with Lombok before, that's **much** neater than I feared. Actually, looking through the whole Lombok feature list, it has a lot of cool stuff.

Would also make uninstalling CaptainsLog and restoring the delegated intel much cleaner. Hrmmm...
